Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10
+
+RUN apt update
+
+RUN mkdir /my_market
+
+WORKDIR /my_market
+
+COPY ./src ./src
+COPY ./requirements.txt ./requirements.txt
+
+RUN python -m pip install --upgrade pip
+RUN pip install -r ./requirements.txt
+
```

Comment:
Use USER command: You can use the USER command to run your application with a non-root user, which is more secure.

Reply:
USER app

CMD ["python", "src/manage.py", "runserver", "0:8008"]